packages/get-platform/src/__tests__/parseDistro.test.ts

```diff
@@ -1,35 +1,48 @@
 import { parseDistro } from '../getPlatform'
 
-const tests = [
-  {
-    name: 'simple',
-    content: `
+describe('parseDistro', () => {
+  const tests = [
+    {
+      name: 'debian, ID',
+      content: `
 ID=debian
-  `,
-    expect: 'debian',
-  },
-  {
-    name: 'simple with quotes',
-    content: `
+      `,
+      expect: {
+        targetDistro: 'debian',
+        familyDistro: 'debian',
+        originalDistro: 'debian',
+      },
+    },
```

Comment:
The tests are essentially the same one a before, with an extended returned value and additional test cases (e.g., `unknown` as an `orginalDistro`)